src/components/FilterDrinks.spec.js

```diff
@@ -21,10 +21,14 @@ describe('FilterDrinks', () => {
       <FilterDrinks handleChangeFilter={radioCallback} currentFilter={'all'} />
     );
 
+    const radioButtonAll = screen.getByLabelText('All');
     const radioButtonAlcoholic = screen.getByLabelText('Alcoholic');
+    const radioButtonNonAlcoholic = screen.getByLabelText('Non-Alcoholic');
 
+    userEvent.click(radioButtonAll);
     userEvent.click(radioButtonAlcoholic);
+    userEvent.click(radioButtonNonAlcoholic);
 
-    expect(radioCallback).toBeCalled();
+    expect(radioCallback).toBeCalledTimes(3);
```

Comment:
Insted of using `toBeCalledTimes()` it would be better to work with `toHaveBeenCalledWith('alcoholic')` for example to test if every radio button is called with the correct event.target.value.
After checking your handleChangeFilter again I saw that you call `handleChangeFilter` in your component with the whole event and I think it would be better to call the function directly with the `event.target.value`. :-)
I hope this explains better what I wanted to suggest. 😅

Reply:
Yes thanks, that is something I never would have think about :-D  I will change this